centreon/tests/e2e/cypress/e2e/Dashboards/07-dashboard-manage-permissions.feature

```diff
@@ -0,0 +1,23 @@
+Feature: As a Centreon User with dashboard administration rights,
```

Comment:
This should have a name

Reply:
what do you mean?

Reply:
You wrote a description but forgot to give your feature a name

Reply:
yeah correct
i think that's something i forgot to add on all dashboard test implementations 🫠 so i might add titles on all of them in this PR, will do that now

Reply:
done 👍